src/pages/lib/auth/emailpassword/q/platform/[platform].mdx

```diff
@@ -1,12 +1,333 @@
+    }
+  })
+}
+
+```
+
+### Sign in
+
+You pass the username and password to the `signIn` method of the Auth class to enable username and password sign-in.
+
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify';
+
+type SignInParameters = {
+  username: string;
+  password: string;
+};
+
+export async function signIn({ username, password }: SignInParameters) {
+  try {
+    const user = await Auth.signIn(username, password);
+  } catch (error) {
+    console.log('error signing in', error);
+  }
+}
+```
+</Block>
+<Block name="JavaScript">
+
+```js
+import { Auth } from 'aws-amplify';
+
+async function signIn() {
+  try {
+    const user = await Auth.signIn(username, password);
+  } catch (error) {
+    console.log('error signing in', error);
+  }
+}
+```
+</Block>
+</BlockSwitcher>
+
+### Sign out
+
+To set up local sign-out you use the `signOut` method of the Auth class.
+
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify';
+
+export async function signOut() {
+  try {
+    await Auth.signOut();
+  } catch (error) {
+    console.log('error signing out: ', error);
+  }
+}
+```
+</Block>
+<Block name="JavaScript">
+
+```js
+import { Auth } from 'aws-amplify';
+
+async function signOut() {
+  try {
+    await Auth.signOut();
+  } catch (error) {
+    console.log('error signing out: ', error);
+  }
+}
+```
+</Block>
+</BlockSwitcher>
+
+You can sign out users from all devices by adding global sign-out. It also invalidates all refresh tokens issued to an user. The user's current access and Id tokens remain valid on other devices until the refresh token expires (access and Id tokens expire one hour after they are issued).
+
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify';
+
+export async function signOut() {
+  try {
+    await Auth.signOut({ global: true });
+  } catch (error) {
+    console.log('error signing out: ', error);
+  }
+}
+```
+</Block>
+<Block name="JavaScript">
+
+```js
+import { Auth } from 'aws-amplify';
+
+async function signOut() {
+  try {
+    await Auth.signOut({ global: true });
+  } catch (error) {
+    console.log('error signing out: ', error);
+  }
+}
+```
+</Block>
+</BlockSwitcher>
+
+Once you implement the Amplify Library methods within your application, you can test the sign-up, sign-in, and sign-out functionality.
+
+### View the created user in the AWS Console
+
+After you test the sign-up, sign-in, and sign-out functionality, you can take a look at the created user in the Amazon Cognito console. To do so, you can use `amplify console auth` to open the console directly from the CLI, or take a look at the existing User Pools in the console. Note the users status should be `CONFIRMED` if you successfully verified the user; however, it can be in `unconfirmed` if the user is not yet verified. You can also check to make sure any other required attributes are updating correctly.
```

Comment:
Should we have"CONFIRMED" in all caps, but "unconfirmed" lower case?

Reply:
Good catch - just reviewed and both should be all caps based on how they are listed in Cognito.

Reply:
```suggestion
After you test the sign-up, sign-in, and sign-out functionality, you can take a look at the created user in the Amazon Cognito console. To do so, you can use `amplify console auth` to open the console directly from the CLI, or take a look at the existing User Pools in the console. Note the users status should be `CONFIRMED` if you successfully verified the user; however, it can be in `UNCONFIRMED` if the user is not yet verified. You can also check to make sure any other required attributes are updating correctly.
```